tasks/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
 - name: Make certain Riak is stopped
```

Comment:
So, @jsmartin is correct in Issue #26 and whenever I add ignore_errors to a task, I shudder a little because it feels like I am just using duct tape to fix a problem. I think if we removed this task, and updated the enabled task to state=restarted, we will get the effect we want of loading the potentially new config and handling for new installs as I just tested (least on Cent7) that if riak is not running, restarted works properly.

Make those edits and I will accept this PR. Thanks man, you have no idea how much I appreciate having an extra set of eyes and help with this repo. :)